app-engine/demo-version-index/main.py

```diff
@@ -0,0 +1,162 @@
+# Shaka Player Version Index - Appspot Entrypoint
+# Copyright 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+# Generate an index of Shaka Player versions on appspot.
+
+import collections
+import google.appengine.api.modules
+import jinja2
+import os
+import random
+import re
+
+from flask import Flask, render_template
+
+DEMO_URL_TEMPLATE = 'https://{0}-dot-shaka-player-demo.appspot.com/'
+
+# In Google Hosted Libraries
+HOSTED_URL_TEMPLATE = 'https://ajax.googleapis.com/ajax/libs/shaka-player/{0}/shaka-player.{1}'
+
+# Before Google Hosted Libraries, v1 appspot URLs
+V1_URL_TEMPLATE = 'https://{0}-dot-shaka-player-demo.appspot.com/shaka-player.{1}'
+# Before Google Hosted Libraries, v2 and v3 appspot URLs
```

Comment:
I feel like this should say "v2 and above appspot URLs"
Unless we are planning on changing the folder structure for v4 and I forgot about it, I guess?